util/install.sh

```diff
@@ -173,7 +199,41 @@ install_curvebs() {
         #   3) strip debug information
         create_project_dir $project_prefix
         copy_file "$binary" "$project_prefix/sbin/$project_bin_filename"
-        get_build_mode && strip_debug_info "$project_prefix/sbin/$project_bin_filename"
+        strip_debug_info "$project_prefix/sbin/$project_bin_filename"
+
+        success "install $project_name success\n"
+    done
+}
+
+
+install_curvefs() {
+    for target in `get_targets`
+    do
+        local regex_target="//(curvefs/src/([^:]+)):(.+)"
+        if [[ ! $target =~ $regex_target ]]; then
+            die "unknown target: $target\n"
+        fi
+
+        local project_name="${BASH_REMATCH[2]}"  # ex: metaserver
+        g_project_name=$project_name
+        local project_prefix="$g_prefix/$project_name"  # ex: /usr/local/curvefs/metaserver
+
+        local project_bin_filename=${BASH_REMATCH[3]}  # ex: curvefs_metaserver
+        local bazel_dir="$g_curve_dir/bazel-bin"
+        local binary="$bazel_dir/${BASH_REMATCH[1]}/$project_bin_filename"
+
+        # The below actions:
+        #   1) create project directory
+        #   2) copy binary to project directory
+        #   3) generate servicectl script into project directory.
+        create_project_dir $project_prefix
+        copy_file "$binary" "$project_prefix/sbin"
```

Comment:
If copying the file fails, for example the file does not exist, it is recomend to print an error and exit directly without continuing to execute.

Reply:
This feature is already included in `create_project_dir`.